src/content/docs/guides/docker-images/cardano-node.md

```
@@ -0,0 +1,47 @@
+
+```sh
+docker run --detach --name cardano-node \
+  -v node-data:/data/db -v node-ipc:/ipc \
+  -e NETWORK=preprod -p 3001:3001 \
+  ghcr.io/blinklabs-io/cardano-node:latest
+```
+
+An empty preprod or mainnet database can be bootstrapped from a Mithril snapshot.
+Keep `/data/db` persistent and set `RESTORE_SNAPSHOT=false` to disable this.
+
+### Runtime variables
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)